app/controllers/comments_controller.rb

```diff
@@ -9,8 +16,24 @@ def create
     @user = current_user
     @post = Post.find(params[:post_id])
     @comment = @post.comments.new(author: @user, post: @post, text: params[:comment][:text])
-    @comment.save
-    redirect_to user_post_path(@post.author, @post)
+    # redirect_to user_post_path(@post.author, @post)
+    # respond_to do |format|
+    if @comment.save
+      format.html do
+        redirect_to user_post_path(@post.author, @post)
+      end
+      format.json do
+        render json: @comment
+      end
+    else
+      format.html do
+        { render: new }
+      end

```

Comment:
- [ ] You're also required to create an API endpoint to add a comment to a post. This has not been implemented. Kindly attend to it.

### Project Requirement
![add-comment](https://user-images.githubusercontent.com/26597432/232112270-00f460a5-999b-4663-8005-c68dd65226a9.png)

Reply:
Thank you @Qoosim for pointing out the short coming. I worked it out after your suggestion.